common/djangoapps/student/management/commands/bulk_change_enrollment_csv.py

```diff
@@ -0,0 +1,98 @@
+"""
+Management command to change many user enrollments in many courses using
+csv file.
+"""
+import csv
+import logging
+from os import path
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from student.models import CourseEnrollment, User
+from student.models import CourseEnrollmentAttribute
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Command(BaseCommand):
+    """
+        Management command to change many user enrollments in many
+        courses using the csv file
+        """
+
+    help = """
+        Change the enrollment status of all the users specified in
+        the csv file in the specified course to specified course
+        mode.
+        Could be used to update Effected users by order
+        placement issues. If number of students is a lot for the
+        different courses.
```

Comment:
Please rephrase it as "If there are large number of effected students in different courses."